CONTRIBUTING.md

```diff
@@ -1,53 +1,121 @@
-    **Steps to create a postgresql db on your system**
-        1. `sudo su - postgres`
-        2. `psql`
-        3. `CREATE DATABASE <db-name>;`
-        4. `CREATE USER <db-username> WITH PASSWORD <db-password>;`
-        5. `ALTER ROLE <DB-USERNAME> SET client_encoding TO 'utf-8';`
-        6. `GRANT ALL PRIVILEGES ON DATABASE <db-name> TO <db-username>;`
-
-4. You can have the project running in a virtual environment or otherwise. Virtual environment is a preffered option.
-    **Steps to set up a virtual environment for this project**
-    1. `python3 -m venv myenv`
-
-    2. Activate the virtual environment `source myvenv/bin/activate`
-
-    3. Install required modules `pip install -r requirements.txt`
-
-5.  Store your secrets by adding them at the end of `env/bin/activate`. They will be set whenever you run the virtual environment.
-
-    ```bash
-    export SECRET_KEY=''
-    export DB_NAME=''
-    export DB_USER=''
-    export DB_PASS=''
-    ```
+    -----------------------------OR------------------------------------------    
+     `(Note:- Below method is preferrable if you are on windows,   but works well for both linux/Windows.)  `
+    we can use a external file to store all the senstive data ( like `SECRET_KEY`, DB_PASS, DB_USER).   
+
+    Handling and Accessing sensitive data,   
+      To access sensitve data from  a different file: -  
+
+        1. `pip install python-decouple` ( Already added in requirements.txt)   
+        2. create `.env` file. (in the root folder where manage.py is present)   
+        3. Enter the following in .env file ( fill your sensitive data without string quotes.),  
+            SECRET_KEY = 'secretkey'  
+            DEBUG = True  
+            DB_NAME = 'db-name'  
+            DB_USERNAME = 'db-username'  
+            DB_PASSWORD = 'password'  
+            DB_HOST = 'localhost'  
+
+        4. Now open `settings.py` and import   
+                `from decouple import config`   ( By Default, already present.)   
+
+        5. Type, `SECRET_KEY = config("SECRET_KEY")`  
+            and `DEBUG = config('DEBUG')`      ( By Default, already present.)  
+
+        6. In DATABASES, Replace this  or just Comment out those already present in `settings.py` file.  
+
+            DATABASES = {   
+                'default': {  
+                    'ENGINE': 'django.db.backends.postgresql_psycopg2',  
+                    'NAME': config('DB_NAME'),  
+                    'USER': config('DB_USERNAME'),  
+                    'PASSWORD': config('DB_PASSWORD'),  
+                    'HOST': config('DB_HOST'),  
+                    'PORT': '',  
+                }  
+            }  
+
+        7. Since, .env file conatins all of sensitive data. Before uploading, add `.env` file to gitignore to prevent sensitve data uploading on
+        github.  
+            To add, just open `.gitignore`, add `.env`. Save file.   ( By Default, already present.)  
+
+
```

Comment:
Remove this we don't want to confuse people, only one way is enough.

Reply:
Done